lib/extensions.js

```diff
@@ -159,18 +159,13 @@ module.exports = function(realmConstructor) {
             }
         }
 
-        realmConstructor.prototype.subscribeToObjects = function(objectType, query) {
-            const realm = this;
-            let promise = new Promise((resolve, reject) => {
-                realm._subscribeToObjects(objectType, query, function(err, results) {
-                    if (err) {
-                        reject(err);
-                    } else {
-                        resolve(results);
-                    }
-                });
-            });
-            return promise;
+        // Keep these value in sync with subscription_state.hpp
+        realmConstructor.Sync.SubscriptionState = {
+            Undefined: -3,         // Unknown which state Partial Sync is in.
```

Comment:
Are these going to be exposed to end-users? If yes, `unknown which state Partial Sync is in` is too vague.

Reply:
Agree, we should document them more carefully them in the API documentation.